packages/blocks/src/class-blocks.php

```diff
@@ -104,29 +113,29 @@ public static function remove_extension_prefix( $extension_name ) {
 	 * php if the Gutenberg plugin is not installed, if we know which minimum WP release has the required version we can
 	 * optionally fall back to that.
 	 *
-	 * @param array  $version_requirements An array containing the required Gutenberg version and, if known, the WordPress version that was released with this minimum version.
-	 * @param string $slug The slug of the block or plugin that has the gutenberg version requirement.
+	 * @since 9.0.0
 	 *
-	 * @since 8.3.0
+	 * @param array  $version_requirements An array containing the required Gutenberg version and, if known, the WordPress version that was released with this minimum version.
```

Comment:
The WordPress doc standards seem to say this should be documented something like
```suggestion
	 * @param array  $version_requirements {
	 *     An array containing the required Gutenberg version and, if known, the WordPress version that was released with this minimum version.
	 *
	 *     @type string $gutenberg Gutenberg version.
	 *     @type string $wp        Optional. WordPress version.
	 * }
```